Dockerfile.dapper

```diff
@@ -1,11 +1,9 @@
-FROM ubuntu:18.04
-# FROM arm=armhf/ubuntu:16.04 arm64=arm64v8/ubuntu:16.04
+FROM registry.suse.com/bci/golang:1.16
```

Comment:
The `go.mod` file in this repo is 1.14. Do we want to bump that too? Or put this at 1.14?

Reply:
We can bump it. I'll extend the PR with this 👍🏻